src/test/java/uk/gov/hmcts/cmc/claimstore/services/MediationCSVServiceTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.cmc.claimstore.repositories.CaseRepository;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim.getClaimWithFullDefenceWithMediation;
+
+@RunWith(MockitoJUnitRunner.class)
+public class MediationCSVServiceTest {
+
+    private static final String AUTHORISATION = "Bearer: aaa";
+
+    private MediationCSVService mediationCSVService;
+
+    @Mock
+    private CaseRepository mockCaseRepository;
+
+    private List<Claim> mediationClaims = new ArrayList<>();
+
+    @Before
+    public void setUp () {
```

Comment:
Our Java style does not separate method names from parameter lists.

Reply:
I assume you are talking about space between `setUp ()`!!, its corrected now